deploy/README.md

```diff
@@ -23,15 +32,6 @@ Set an environment variable named `NGC_CLI_API_KEY` to your API Key.
 export NGC_CLI_API_KEY=$(cat ~/API_KEY)
 ```
 
-### Login to nvcr.io
-To pull the containers used in the docker compose configuration, you will need to login to `nvcr.io` with your NGC API Key.
-
-```bash
-$ docker login nvcr.io
-Username: $oauthtoken
-Password: <PASTE_API_KEY_HERE>
-```
-
```

Comment:
This section should stay in the README.  The docker compose up will fail to pull containers if not logged in to nvcr.io.

Reply:
Looks like this section was added to main after I forked the repo. Will add it back in.

Reply:
Good point. I think I had a PR merged for most of these changes since your fork.  Thanks!

Reply:
fixed in most recent commit